Makefile

```diff
@@ -222,11 +222,28 @@ generate:
 	controller-gen object:headerFile="hack/boilerplate.go.txt" paths="$(GO_PACKAGES)"
 .PHONY: generate
 
+HELM_CHANNEL ?=latest
+HELM_CHARTS=scylla-operator scylla-manager scylla
+
+# Build Helm charts and publish them in Development GCS repo
+dev-helm-publish: export REPOSITORY ?=https://scylla-operator-charts-dev.storage.googleapis.com/$(HELM_CHANNEL)
+dev-helm-publish: export BUCKET ?=gs://scylla-operator-charts-dev/$(HELM_CHANNEL)
+dev-helm-publish:
+	@$(MAKE) helm-publish
+.PHONY: dev-helm-publish
+
 # Build Helm charts and publish them in GCS repo
-helm-release:
-	mkdir -p $(HELM_LOCAL_REPO)
-	gsutil rsync -d $(HELM_BUCKET) $(HELM_LOCAL_REPO)
-	helm package helm/scylla-operator helm/scylla helm/scylla-manager -d $(HELM_LOCAL_REPO)
-	helm repo index $(HELM_LOCAL_REPO) --url $(HELM_REPOSITORY) --merge $(HELM_LOCAL_REPO)/index.yaml
-	gsutil rsync -d $(HELM_LOCAL_REPO) $(HELM_BUCKET)
-.PHONY: helm-release
+helm-publish: export LOCAL_REPO ?=helm/repo/$(HELM_CHANNEL)
+helm-publish: export BUCKET ?=gs://scylla-operator-charts/$(HELM_CHANNEL)
+helm-publish: export REPOSITORY ?=https://scylla-operator-charts.storage.googleapis.com/$(HELM_CHANNEL)
```

Comment:
I think I'd still make some of these global. late evaluation (`=`) will help you easily override them on the other target while the rest resolves correctly. You can then avoid a global export by doing just the export part in this target